pcsx2/gui/AppConfig.cpp

```diff
@@ -116,144 +120,134 @@ namespace PathDefs
 
 	//The installer installs the folders which are relative to AppRoot (that's plugins/langs)
 	//  relative to the exe folder, and not relative to cwd. So the exe should be default AppRoot. - avih
-	const wxDirName& AppRoot()
+	fs::path AppRoot()
 	{
-		//AffinityAssert_AllowFrom_MainUI();
-/*
-		if (InstallationMode == InstallMode_Registered)
-		{
-			static const wxDirName cwdCache( (wxDirName)Path::Normalize(wxGetCwd()) );
-			return cwdCache;
-		}
-		else if (InstallationMode == InstallMode_Portable)
-*/		
 		if (InstallationMode == InstallMode_Registered || InstallationMode == InstallMode_Portable)
 		{
-			static const wxDirName appCache( (wxDirName)
-				wxFileName(wxStandardPaths::Get().GetExecutablePath()).GetPath() );
-			return appCache;
+			fs::path path(Path::GetExecutableDirectory());
+			path.make_preferred();
+			return path;
 		}
 		else
-			pxFail( "Unimplemented user local folder mode encountered." );
-		
-		static const wxDirName dotFail(L".");
+			pxFail("Unimplemented user local folder mode encountered.");
+
+		static const std::string dotFail(".");
 		return dotFail;
 	}
 
     // Specifies the main configuration folder.
-    wxDirName GetUserLocalDataDir()
+    fs::path GetUserLocalDataDir()
     {
-        return wxDirName(wxStandardPaths::Get().GetUserLocalDataDir());
+        fs::path temp = wxStandardPaths::Get().GetUserLocalDataDir().ToStdString();
+        return temp;
     }
 
 	// Fetches the path location for user-consumable documents -- stuff users are likely to want to
 	// share with other programs: screenshots, memory cards, and savestates.
-	wxDirName GetDocuments( DocsModeType mode )
+	fs::path GetDocuments( DocsModeType mode )
 	{
 		switch( mode )
 		{
 #if defined(XDG_STD) || defined(__APPLE__) // Expected location for this kind of stuff on macOS
 			// Move all user data file into central configuration directory (XDG_CONFIG_DIR)
 			case DocsFolder_User:	return GetUserLocalDataDir();
 #else
-			case DocsFolder_User:	return (wxDirName)Path::Combine( wxStandardPaths::Get().GetDocumentsDir(), pxGetAppName() );
+			case DocsFolder_User:	return Path::Combine(Path::FromWxString(wxStandardPaths::Get().GetDocumentsDir()), Path::FromWxString(pxGetAppName()));
 #endif
 			case DocsFolder_Custom: return CustomDocumentsFolder;
 
-			jNO_DEFAULT
-		}
+ 			jNO_DEFAULT
```

Comment:
Space?

Reply:
So this line was marked resolved when it actually wasn't. Might be worth checking over the other resolved comments in case they have been wrongly marked as resolved as well.

Reply:
Odd. I'll do a quick sweep